programs/rewards/src/state/mining.rs

```diff
@@ -32,6 +32,9 @@ pub struct WrappedImmutableMining<'a> {
     pub weighted_stake_diffs: &'a MiningWeightedStakeDiffs,
 }
 
+pub const ACCOUNT_TYPE_BIT: usize = 0;
```

Comment:
Right now it's byte, not bit. But using bits for that kind of bools makes sense as we're introducing several restrictions. Wdyt of using a bitmap?

Reply:
It's a bitmap already. If you mean a dedicated structure from some crate, I'm not sure it may be used with `Pod` because bool is an u8 in fact and bool is not `Pod`.